content/en/tracing/service_catalog/setup.md

```diff
@@ -67,6 +67,26 @@ As an alternative to the GitHub integration and Terraform, you can use an open-s
 
 This GitHub Action allows you to register your services with the Service Catalog using a GitHub Action, with full control over when this information is sent to Datadog, and implement other compliance checks unique to your organization.
 
+
+## Import Data from Backstage 
+
+If you already have data or services registered in Backstage, you can import these services into Datadog directly. 
+
+If you use API / Terraform, replace the content of YAML content in your requests. 
+If you use GitHub integration, directly save your Backstage YAMLs to a repo with Datadog read permission. Note that the file must be located at the root folder of a repo, and the file must be named `service.datadog.yaml`. 
+
+
+It’s a partial translation, specifically, 
```

Comment:
What is a partial translation? Please make this a complete sentence :)

Reply:
How about 'Datadog only recognize selected fields in Backstage YAMLs, specifically,